src/util/value.ts

```diff
@@ -80,6 +80,20 @@ const valueTypeMapper: Partial<Record<GQLValueType, ValueType>> = {
 
 export type ValueType = keyof ValueTypes;
 
+const enumTypeMapper: Partial<Record<any, string>> = {
+  [Theme.Default]: "default",
+  [Theme.Dark]: "dark",
+  [Theme.Light]: "light",
+  [AssetSortType.Date]: "date",
+  [AssetSortType.Name]: "name",
+  [AssetSortType.Size]: "size",
+};
+
+export const toGQLEnum = (val?: any) => {
+  if (!val) return;
+  return (Object.keys(enumTypeMapper) as any[]).find(k => enumTypeMapper[k] === val);
+};
+
```

Comment:
This function uses any, which may increase the number of bugs not found by compiler type checking. Implement the conversion function for each enum rather than loosening the type.

Reply:
Oh okay. I didn't think it was really DRY to make a conversion function for each enum (for now there is only two, but later we could have more, multiplying the code). 
I guess, thinking about the rest of our typescript code, that it makes sense strong types > DRY code. 
I'll update now!